### PR TITLE
Add django-nose recipe

### PR DESCRIPTION
**Pull request by sodre**
django-nose provides all the goodness of nose in your Django tests.

### CODE REVIEW

File: recipes/django-nose/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - django_nose
+    - django_nose.management
+    - django_nose.management.commands
+
+about:
+  home: http://github.com/django-nose/django-nose
+  license: BSD License
```
Comment (jakirkham) [inline recipes/django-nose/meta.yaml]: Please drop `License` and specify number of clauses. Believe this is `BSD 3-Clause`.

File: recipes/django-nose/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - nose >=1.2.1
```
Comment (jakirkham) [inline recipes/django-nose/meta.yaml]: Is `nose` needed at build time or can it be dropped here?

Comment (sodre) [inline recipes/django-nose/meta.yaml]: You are right. It builds without it.

File: recipes/django-nose/meta.yaml (hunk 1 above)
Comment (sodre) [inline recipes/django-nose/meta.yaml]: Corrected.

**Comment by jakirkham** [comment]
Thanks @sodre. LGTM.

**Pull request merged**